lib/zendesk_api/middleware/response/sanitize_response.rb

```diff
@@ -0,0 +1,21 @@
+module ZendeskAPI
+  module Middleware
+    module Response
+      class SanitizeResponse < Faraday::Response::Middleware
+        def on_complete(env)
+          to_utf8!(env[:body]) unless env[:body].strip.empty?
+        end
+
+        def to_utf8!(str)
+          str.force_encoding(Encoding::UTF_8)
+          if ''.respond_to?(:scrub)
+            str.scrub!('') unless str.valid_encoding?
+          else
+            str.encode!(Encoding::UTF_16, :invalid => :replace, :replace => '')
```

Comment:
most of our projects are on 2.1 ... maybe that's good enough ?
otherwise whatever works ...